src/Common/Constants.ts

```diff
@@ -339,7 +339,6 @@ export enum ConflictOperationType {
 }
 
 export enum ConnectionStatusType {
-  Connecting = "Connecting",
   Allocating = "Allocating",
```

Comment:
>Allocating [](http://example.com/codeflow?start=16&length=10)

use Connecting instead of Allocating

Reply:
removed allocating but it was not showing updated changes